Makefile

```diff
@@ -199,7 +199,7 @@ coverage:
 
 .PHONY: build clean get-tools test check \
 	cover htmlcover coverage \
-	install install-bin install-conf install-datadir install-demo install-identity-scripts \
-	install-inventory-scripts install-modules install-modules-gen install-systemd \
+	install install-bin install-conf install-datadir install-identity-scripts \
```

Comment:
This is growing quite big an unruly I think. What if we added a `.PHONY` to each target instead? It is quite verbose, and I know I'm basically the only proponent north of equator it seems. What do you think?

Reply:
I don't understand what it means to be a "proponent north of equator". Other than that I agree and I am putting all of them in individual `.PHONY` targets